tests/cases/care_partner_platform/case_care_platform_participants.yaml

```diff
@@ -3948,4 +3948,172 @@ TestCarePartnerParticipantInfoEditOrAddEmergencyRoomAdmissionsInfo:
       Id: $PAGE.care_platform_edit_participant_hospitalization.delete_emergency_room_admission_dialog_title$
     - Type: wait_for
       Strategy: xpath
-      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
\ No newline at end of file
+      Id: $PAGE.care_platform_notifications.emergency_room_admission_deleted_notification$
+
+#CXTB-111 Participant Profile - Review, Edit or Add Vendor Info
+TestCarePartnerParticipantReviewEditOrAddVendorInfo:
+  Environment: Settings
+  Roles:
+    - Role: desktopChrome
+      App: desktop
+  Actions:
+  # Calling Login case.
+    - Type: case
+      Value: CarePartnerLogin
+  # looking for a senior profile and click on it
+    - Type: click
+      Strategy: xpath
+      Id: $PAGE.care_platform.navigation_participant$
+    - Type: click 
+      Strategy: xpath
+      Id: $PAGE.care_platform_home.first_call_of_queue$
```

Comment:
it shouldn't be "first call", this is a participant, not a call

Reply:
I' m just reused the page object.